sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -890,138 +883,6 @@ def test_online_retrieval_with_go_server(
     )
 
 
-@pytest.mark.skip
-@pytest.mark.integration
-@pytest.mark.goserver
-def test_online_store_cleanup_with_go_server(go_environment, go_data_sources):
```

Comment:
Why do we have to delete these tests? They're the only tests currently working that tests the go feature server.

Reply:
What's the purpose of this tests? It doesn't check anything go server related. This is just duplicate of `test_online_store_cleanup`

Reply:
`test_go_server_life_cycle ` was testing process management. There's no process anymore

Reply:
Oh right I didn't actually check, I was under the impression that we only had 2 go online tests but we had 3, I was mainly worried about online_retrieval_with_go_server but that is still there!